src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -415,10 +415,25 @@ function twentytwelve_content_nav( $html_id ) {
 		global $wp_query;
 
 		if ( $wp_query->max_num_pages > 1 ) : ?>
+			<?php
```

Comment:
```suggestion
		if ( $wp_query->max_num_pages > 1 ) :
```
This does not need to close and open PHP tags. By removing them, the `<nav id="<?php echo esc_attr( $html_id ); ?>" class="navigation">` line would still indent by three tabs instead of increasing to six.